src/ext/java/org/opentripplanner/ext/emissions/EmissionsModule.java

```diff
@@ -50,10 +50,14 @@ public void buildGraph() {
       double carAvgEmissionsPerMeter = carAvgCo2PerKm / 1000 / carAvgOccupancy;
 
       for (ConfiguredDataSource<GtfsFeedParameters> gtfsData : dataSources.getGtfsConfiguredDatasource()) {
```

Comment:
While you're here: `emissionsData` can be a local variable and all instance variables can be final.